lib/src/mapbox_maps_platform.dart

```diff
@@ -181,3 +187,36 @@ class _MapboxMapsPlatform {
     }
   }
 }
+
+/// A registry to hold suffixes for Channels.
+///
+class _SuffixesRegistry {
+  _SuffixesRegistry._instance();
+
+  int _suffix = -1;
+  final Set<int> suffixesInUse = {};
+  final Set<int> suffixesAvailable = {};
+
+  int getSuffix() {
```

Comment:
is it thread safety?

Reply:
not sure how it works in flutter but imo the whole class should be completely thread-safe

Reply:
It's internal util class and is always called from single thread, so not a concern here